scripts/exclude-activity-gen.json

```diff
@@ -49,5 +49,10 @@
   "EspressoTestActivity",
   "FragmentBackStackActivity",
   "ChildFragmentMapInDialogActivity",
-  "PerformanceMeasurementActivity"
+  "PerformanceMeasurementActivity",
```

Comment:
Is there a reason **why** are they failing?
Perhaps create a task to track that?

Reply:
Yes, we have. Actually I have a draft pr for this https://github.com/mapbox/mapbox-gl-native-internal/pull/765

Reply:
Cool.
We need not to forget to remove those tests from exclude list.